enterprise/frontend/src/metabase-enterprise/audit_app/components/AuditParameters.jsx

```diff
@@ -1,29 +1,24 @@
 import React from "react";
+import PropTypes from "prop-types";
 
 import _ from "underscore";
 
 const DEBOUNCE_PERIOD = 300;
 
-type AuditParameter = {
-  key: string,
-  placeholder: string,
-};
-
-type Props = {
-  parameters: AuditParameter[],
-  children?: (committedValues: { [key: string]: string }) => React.Element,
-};
-
-type State = {
-  inputValues: { [key: string]: string },
-  committedValues: { [key: string]: string },
+const propTypes = {
+  parameters: PropTypes.arrayOf(
+    PropTypes.shape({
+      key: PropTypes.string.isRequired,
+      placeholder: PropTypes.string.isRequired,
+    }),
+  ),
+  children: PropTypes.func,
 };
 
 export default class AuditParameters extends React.Component {
-  props: Props;
-  state: State;
+  static propTypes = propTypes;
```

Comment:
I thought we agreed not to use static propTypes?

Reply:
Oh, fixed that